site/docs/latest/releaseNotes.md

```diff
@@ -0,0 +1,483 @@
+large number of entries to recover when client failures occur. BookKeeper introduces `parallel ledger recovery` in 4.5.0 to allow
+batch recovery to improve ledger recovery speed.
+
+To enable this feature, please set `enableParallelRecoveryRead` to `true` on your clients. You can also set `recoveryReadBatchSize`
+to control the batch size of recovery read.
+
+#### multiple journals
+
+Prior to 4.5.0, bookies are only allowed to configure one journal device. If you want to have high write bandwidth, you can raid multiple
+disks into one device and mount that device for jouranl directory. However because there is only one journal thread, this approach doesn't
+actually improve the write bandwidth.
+
+BookKeeper introduces multiple journal directories support in 4.5.0. Users can configure multiple devices for journal directories.
+
+To enable this feature, please use `journalDirectories` rather than `journalDirectory`.
+
+### Operations
+
+#### LongHierarchicalLedgerManager
+
+Apache BookKeeper supports pluggable metadata store. By default, it uses Apache ZooKeeper as its metadata store. Among the zookeeper-based
+ledger manager implementations, `HierarchicalLedgerManager` is the most popular and widely adopted ledger manager. However it has a major
+limitation, which it assumes `ledger-id` is a 32-bits integer. It limits the number of ledgers to `2^32`.
+
+`LongHierarchicalLedgerManager` is introduced to overcome this limitation.
+
+See [Ledger Manager](../develop/ledger-manager) for more details and learn how to migrate `HierarchicalLedgerManager` to `LongHierarchicalLedgerManager`.
+
+#### Weight-based placement policy
+
+`Rack-Aware` and `Region-Aware` placement polices are the two available placement policies in BookKeeper client. It places ensembles based
+on users' configured network topology. However they both assume that all nodes are equal. `weight-based` placement is introduced in 4.5.0 to
+improve the existing placement polices. `weight-based` placement was not built as separated polices. It is built in the existing placement policies.
+If you are using `Rack-Aware` or `Region-Aware`, you can simply enable `weight-based` placement by setting `diskWeightBasedPlacementEnabled` to `true`.
+
+#### Customized Ledger Metadata
+
+A `Map<String, byte[]>` is introduced in ledger metadata in 4.5.0. Clients now are allowed to pass in a key/value map when creating ledgers.
+This customized ledger metadata can be later on used by user defined placement policy. This extends the flexibility of bookkeeper API.
+
+#### Add Prometheus stats provider
+
+A new [Prometheus](https://prometheus.io/) [stats provider](https://github.com/apache/bookkeeper/tree/master/bookkeeper-stats-providers/prometheus-metrics-provider)
+is introduce in 4.5.0. It simplies the metric collection when running bookkeeper on [kubernetes](https://kubernetes.io/).
```

Comment:
I don't think test improvements need to be in `highlights`.

I will include the tool improvements.